src/DayColumn.js

```diff
@@ -448,4 +383,81 @@ DayColumn.defaultProps = {
   timeslots: 2,
 }
 
+/**
+ * @param {Object} props
+ * @param {() => Date} props.getNow
+ * @param {boolean=} props.isNow
+ * @param {Date} props.max
+ * @param {Date} props.min
+ * @param {Object} props.slotMetrics
+ */
+function TimeIndicator({ getNow, isNow, max, min, slotMetrics }) {
+  const [position, setPosition] = useState(0)
```

Comment:
could totally be a nit, if it's at 0 and then we set the position, could there be a jump if the parent have animation? Would it be better to have a state to track if should show and only show once we know the position?